src/main/public/assets/scripts/validations/ca_reviewrankedsuppliers.js

```diff
@@ -16,7 +19,15 @@ document.addEventListener('DOMContentLoaded', () => {
         $("#ca_rank_summary_list").html('<li><a href="#">Please select the suppliers</a></li>');
         $('html, body').animate({ scrollTop: 0 }, 'fast');
     }
-    else if($('input[type=checkbox]:checked').length > 0 && $('#ca_justification').val()==='')
+    else if($('input[type=checkbox]:checked').length > 0 && $('input[type=checkbox]:checked').length < matches[0])
+    {
+        e.preventDefault();
+        $('#ca_reviewrankedsuppliers_error_summary').removeClass('hide-block');
```

Comment:
1. Use common function to display error summary. ccsZPresentErrorSummary(errorStore);
2. errorStore.push(["tag_ID","Error Message"])

Reply:
resolved